src/MainStack.ts

```diff
@@ -312,16 +314,46 @@ export class MainStack extends Stack {
       );
       return;
     }
-    for (const openKlantRegistrationService of this.configuration.openKlantRegistrationServices) {
-      new OpenKlantRegistrationService(this, openKlantRegistrationService.cdkId, {
-        api: api.api,
-        openKlantRegistrationServiceConfiguration: openKlantRegistrationService,
-        criticality: this.configuration.criticality,
-        key: this.key,
-      });
+    for (const openKlantRegistrationService of this.configuration
+      .openKlantRegistrationServices) {
+      new OpenKlantRegistrationService(
+        this,
+        openKlantRegistrationService.cdkId,
+        {
+          api: api.api,
+          openKlantRegistrationServiceConfiguration:
+            openKlantRegistrationService,
+          criticality: this.configuration.criticality,
+          key: this.key,
+        },
+      );
     }
   }
 
+  private openProductServices(api: ApiGateway, platform: ContainerPlatform) {
+    if (!this.configuration.openProductServices) {
+      console.warn('No openProduct configuration provided. Skipping creation!');
+      return;
+    }
+    new OpenProductService(this, 'open-product', {
+      hostedzone: this.hostedzone,
+      key: this.key,
+      cache: this.cache,
+      cacheDatabaseIndex: 11,
```

Comment:
Als we er nu meer hebben gaat dit wel fout denk ik? (overigens op alle andere plekken ook)